tests/js/spec/utils/projects.spec.jsx

```diff
@@ -496,5 +503,22 @@ describe('utils.projects', function() {
       // expect the store action to be called
       expect(loadProjects).toHaveBeenCalledWith(mockProjects);
     });
+
+    it('does not refetch projects that are already loaded in the store', async function() {
+      ProjectsStore.loadInitialData(mockProjects);
+
+      const wrapper = createWrapper({allProjects: true});
+      wrapper.update();
+
+      expect(renderer).toHaveBeenCalledWith(
+        expect.objectContaining({
+          fetching: false,
+          isIncomplete: null,
+          hasMore: false,
+          projects: mockProjects,
+        })
+      );
+      expect(request).not.toHaveBeenCalled();
```

Comment:
Should we make sure the action wasn't called? I guess the request may be enough

Reply:
never can be too safe, just added it in.